specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NspLinkPut.json

```diff
@@ -0,0 +1,69 @@
+{
+  "title": "Create/Update NSP link",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkSecurityPerimeterName": "nsp1",
+    "linkName": "link1",
+    "parameters": {
+      "properties": {
+        "autoApprovedRemotePerimeterId": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityPerimeters/nsp2"
```

Comment:
>autoApprovedRemotePerimeterId [](http://example.com/codeflow?start=9&length=29)

are you including linked access checks to ensrure that user has permissions on this resource ?

Reply:
Yes, user should have Microsoft.Network/networkSecurityPerimeters/linkPerimeter/action permission on this resource